README.md

```diff
@@ -377,6 +375,24 @@ Add the following environment variable on Netlify:
 12. In `Triggers` select `A document is published` and `A document is unpublished`
 13. Click `Add this webhook`
 
+## User Roles
+
+All site users have one or more roles assigned to them. The role determines what actions the user can take on the site.
+
+As soon as a user is signed in, the system assign a `subscriber` role as default. Role assignment is handled manually by the site administrators.
+
+**The roles are:**
+
+| User Role                     | Permissions                                                                                                                             |
+|-------------------------------|----------------------------------------------------------------------------------------------------------------------------------------|
+| `subscriber`                  | This is the default role assigned to all users. It allows the user to subscribe to new incidents, specific incidents, entities, and anything else that is subscribeable. |
+| `submitter`                   | This role allows the user to submit new incidents under their user account.                                                            |
+| `incident_editor`             | This role allows the user to:<br>- Edit and clone incidents<br>- See the live incident data<br>- Add, edit, approve and delete Variants<br>- View and submit incident candidates<br>- Restore previous versions of incidents and reports. |
```

Comment:
Maybe add a quick explanation of what we mean with "live incident data"?

Reply:
Added